Tests/ClientRuntimeTests/NetworkingTests/Streaming/BufferedStreamTests.swift

```diff
@@ -6,65 +6,154 @@
 //
 
 import XCTest
-@testable import ClientRuntime
+import ClientRuntime
 
 final class BufferedStreamTests: XCTestCase {
```

Comment:
Cleaned up this test class and added some new tests for the new async methods I added to this stream.